### PR TITLE
Improve commerce section

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | 4.2, master

Improves and fixes product/catalog doc based on feedback. Also, changes labels in navigation to fit feature naming.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date

### CODE REVIEW

File: docs/shop_administration/manage_products.md [hunk 1]
```diff
@@ -21,24 +21,46 @@ A product type can be, for example, a sofa or a keyboard.
...
+1\. Go to **Product catalog** -> **Product Types** and click **Create**.
+
 ![Adding a product type](img/create_product_type.png)
 
+2\. Fill in basic product information: name, identifier and description.
+
 Each product has a product code that must be unique. It identifies the product in the system.
 Product code can have up to 64 characters. It can contain only letters, numbers, underscores, and dashes.
 
+3\. If needed, add additional Fields decriptinb the product.
```
Comment (juskora) [inline docs/shop_administration/manage_products.md:35]:
```suggestion
3\. If needed, add additional Fields describing the product.
```

File: docs/shop_administration/manage_products.md [hunk 2]
```diff
@@ -21,24 +21,46 @@ A product type can be, for example, a sofa or a keyboard.
...
 
 Typical product attribute examples are: length, weight, color, format, and so on.
 
+Attribute types define what type of information you can store in an attribute.
+Available attribute types are:
+
+- Checkbox
+- Color - presented as a hex value
+- Float - represents a number with fractions
+- Integer - represents a number without fractions
```
Comment (juskora) [inline docs/shop_administration/manage_products.md:52]:
```suggestion
- Integer - represents a number without fractions (a whole number)
```

File: docs/shop_administration/manage_products.md [hunk 3]
```diff
@@ -21,24 +21,46 @@ A product type can be, for example, a sofa or a keyboard.
...
+- Measurement (single) - measurement with a single value in given unit
+- Selection - one of a list of customizable options
+
 Each attribute belongs to an attribute group.
 An example of an attribute group can be dimensions (which consists of length, width, height, and so on).
 
 ![Creating an attribute in a group](img/create_attribute.png)
 
 When adding attributes to product types, you can add both individual attributes, and whole groups.
+You can also remove attributes from groups that you do not want to use for the given product type.
```
Comment (juskora) [inline docs/shop_administration/manage_products.md:63]: It's just a suggestion. What I think is missed, the instruction how we can remove attributes and that we have to drag and drop attributes.

File: docs/shop_administration/manage_products.md [hunk 4]
```diff
@@ -51,12 +73,14 @@ Attributes that have this option are used to [automatically generate product var
...
 ## Product availability and stock
 
-To control a product's availability and the available stock, use the product's **Availability** tab.
+To control a product's availability and the available stock, in product or variant view go to **Availability** tab.
 If a product can have variants, you must set availability for each variant separately.
 
 ![Product availability](img/product_availability_tab.png)
 
-Use the switcher to indicate whether the product is currently available in the catalog.
+Click **Create** to add availability to a product that doesn't have it, or **Edit** to modify existing availability settings.
```
Comment (juskora) [inline docs/shop_administration/manage_products.md:81]:
```suggestion
Click **Create** to add availability to a product that does not have it, or **Edit** to modify existing availability settings.
```

Comment (juskora) [reply docs/shop_administration/manage_products.md:81]: There are 2 Edit buttons. I think we should add that the button with edit icon must be clicked to edit the availability.

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/shop_administration/manage_products.md (hunk 4 above)
Comment (DominikaK) [reply docs/shop_administration/manage_products.md]: Also added better screenshot which doesn't crop out the button

**Review by juskora — APPROVED** [review]

**Pull request merged**